web/source/osk/visualKeyboard.ts

```diff
@@ -1226,29 +1070,16 @@ namespace com.keyman.osk {
           //layerElement.style.visibility='hidden';
         }
       }
-
-      if(device.touchable) {
-        // Identify and save references to the language key, hide keyboard key, and space bar
-        this.lgKey=this.getSpecialKey(nLayer,'K_LOPT');     //TODO: should be saved with layer
-        this.hkKey=this.getSpecialKey(nLayer,'K_ROPT');
-      }
-
-      // Define for both desktop and touchable OSK
-      this.spaceBar=this.getSpecialKey(nLayer,'K_SPACE'); //TODO: should be saved with layer
     }
 
     /**
      * Adjust the absolute height of each keyboard element after a rotation
      *
      **/
-    adjustHeights(height: number) {
+    refreshLayout(height: number) {
```

Comment:
Intentional method rename - adjusting element heights is _part_ of the process of refreshing the layout.

Reply:
Note from development of the successor PR (#5459):  this method, _**as seen here**_, only handles touch-enabled layout logic.  (Emphasis because that will be changing with #5459.)

That `height` parameter is what configures the touch keyboard to use the correct height.  Meanwhile, desktop form-factors set the keyboard's size through the `setSize` function, which is not affected by this PR.